mne/tests/test_report.py

```diff
@@ -212,6 +212,16 @@ def test_render_mri_without_bem():
 
 
 @testing.requires_testing_data
+@requires_mayavi
+def test_render_sensitivity_map():
+    """Test rendering sensitivity maps."""
+    report = Report(subjects_dir=subjects_dir)
+    pattern=['*sample_audvis-meg-eeg-oct-6-fwd.fif']
+    report.parse_folder(data_path=data_dir, pattern=pattern)
+    assert_equal(['forward', 'sensitivity'], report.sections)
```

Comment:
Probably wrong data_path. and second failure due to extra plot(hence fname) of sensitivity map section. I'll correct this and add suggested test